src/cli.rs

```diff
@@ -96,6 +96,10 @@ pub struct Cli {
     /// Optional key-value pairs to be included in the request.
     #[structopt(name = "REQUEST_ITEM")]
     pub request_items: Vec<RequestItem>,
+
+    /// Skip the host's SSL certificate verification
```

Comment:
As you've correctly implemented, the flag isn't only used to disable certificate verification, but possibly also to use an alternative CA bundle. It'd be nice if this were documented.

Reply:
Added!